commands.js

```diff
@@ -1,14 +1,14 @@
 import { getRPSChoices } from './game.js';
-import { capitalize, DiscordRequest } from './utils.js';
+import { capitalize, DiscordRequest } from './utils/utils.js';
 
 export async function HasGuildCommands(appId, guildId, commands) {
   if (guildId === '' || appId === '') return;
 
-  commands.forEach((c) => HasGuildCommand(appId, guildId, c));
+  commands.forEach((c) => VerifyGuildCommand(appId, guildId, c));
```

Comment:
nit: it's all over the place, but our casing is inconsistent. I'd prefer if we stick to one, let's say camelCase, instead of having both camelCase and PascalCase.